woocommerce/woocommerce
# Remove onboarding.js from e2e tests

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This is a follow up PR to [#40256](https://github.com/woocommerce/woocommerce/pull/40256) and removes the `onboarding.js` file from the e2e tests.



### How to test the changes in this Pull Request:

Testing was all conducted in the previous PR.  As a sanity check, just ensure that the e2e tests pass on CI



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
